wallet/src/main/AndroidManifest.xml

```diff
@@ -49,7 +50,11 @@
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="openwallet" android:host="callback" />
+                <data android:scheme="openwallet"/>
+                <data android:host="callback"/>
+                <!-- credential offer / haip -->
+                <data android:scheme="openid-credential-offer"/>
```

Comment:
Any luck figuring out what we'd need to do to have Camera open our app with one of these URLs?

Reply:
I looked into it and it is very puzzling! 🤔 

I tried launching implicit intents via adb and all work just fine. 
I downloaded a QR code scanner app and it scans the QR just fine - tapping on the "open" button launches the intent to our app -- works fine.

A more deeper investigation showed that others are also experiencing similar issues
<img width="532" alt="Screenshot 2024-10-01 at 4 26 07 PM" src="https://github.com/user-attachments/assets/f7b74188-1774-4c0d-ab05-28cbc4af48a0"> https://b.corp.google.com/issues/321657269?pli=1 

And resulted in learning that "**After Android 12 the stock camera of the OS will open the custom schemes as URLs and not as deeplinks.**" https://stackoverflow.com/a/77229265 

The answer seems to be with using App Links https://developer.android.com/training/app-links/#android-app-links where we have an "http/s" URL with a payload that has our custom scheme url.

I think this is also on par with what we may need to do to register our custom scheme via App Links - https://developers.google.com/digital-asset-links/v1/create-statement

Thankfully this issue is only for the camera app :( but it may mean an entire overhaul of how we handle deep links?